packages/modals/stories/4-TooltipModal.stories.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useRef } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { Button } from '@zendeskgarden/react-buttons';
+import { TooltipModal } from '@zendeskgarden/react-modals';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Modals/TooltipModal',
+  component: TooltipModal
+} as Meta;
+
+export const Default: Story = ({
+  placement,
+  hasArrow,
+  zIndex,
+  backdropProps,
+  focusOnMount,
+  restoreFocus,
+  id
+}) => {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const [referenceElement, setReferenceElement] = useState<HTMLButtonElement | null>();
+
+  return (
+    <Grid>
+      <Row>
+        <Col textAlign="center" style={{ padding: 50 }}>
```

Comment:
The `placement` control has been disabled because the new example requires controlled placements. I went ahead and removed the additional spacing as it is now unnecessary.